deck-rest/src/main/kotlin/com/deck/rest/route/ChannelRoute.kt

```diff
@@ -91,8 +91,73 @@ public class ChannelRoute(client: RestClient) : Route(client) {
         body = AddForumThreadReplyReactionRequest(reactionId, teamId)
     )
 
-    public suspend fun getMessage(channelId: UniqueId, messageId: UniqueId): GetMessageResponse = sendRequest<GetMessageResponse, Unit>(
-        endpoint = "/content/route/metadata?route=//channels/$channelId/chat?messageId=$messageId",
-        method = HttpMethod.Get
+    public suspend fun getMessage(channelId: UniqueId, messageId: UniqueId): GetMessageResponse =
+        sendRequest<GetMessageResponse, Unit>(
+            endpoint = "/content/route/metadata?route=//channels/$channelId/chat?messageId=$messageId",
+            method = HttpMethod.Get
+        )
+
+    public suspend fun deleteChannel(teamId: GenericId, channelId: UniqueId, groupId: GenericId? = null): Unit =
+        sendRequest<Unit, Unit>(
+            endpoint = "/teams/$teamId/groups/${groupId ?: "undefined"}/channels/$channelId",
+            method = HttpMethod.Delete
+        )
+
+    public suspend fun getChannelMessages(channelId: UniqueId): GetChannelMessagesResponse =
```

Comment:
Here, and for all the other routes which only have one response parameter, to already return it directly instead of passing the entire response object, for example:
```kotlin
    public suspend fun getChannelMessages(channelId: UniqueId): List<RawMessage> =
        sendRequest<GetChannelMessagesResponse, Unit>(
            endpoint = "/channels/$channelId/messages",
            method = HttpMethod.Get
        ).messages
```